robot/rover/comms_demo/arduino_demos/pong_bing/pong_bing.ino

```diff
@@ -1,26 +1,36 @@
-#include <Arduino.h>
-uint32_t currentTime;
+#include "Arduino.h"
```

Comment:
what is this

Reply:
random leftover test file that someone created a long ass time ago